docker-compose.example.yml

```diff
@@ -5,46 +5,24 @@ services:
     image: unocha/mongo:4.0.5
     volumes:
       - ./db:/srv/db
-    # environment:
-    #   DNSDOCK_NAME: db
-    #   DNSDOCK_IMAGE: hid
   dev:
     image: unocha/nodejs:8.11.3
     volumes:
       - .:/srv/www
     ports:
       - 0.0.0.0:3000:3000
     environment:
-      # DNSDOCK_NAME: api
-      # DNSDOCK_IMAGE: hid
-      # PORT: 80
       NODE_ENV: local
-      ROOT_URL: http://api.hid.vm
-      SMTP_HOST: app.mailhog.vm
+      ROOT_URL: http://localhost:3000
+      SMTP_HOST: mailhog
       SMTP_PORT: 1025
-      SMTP_USER: humanitarianid
-      SMTP_PASS: smtp_password
-      APP_URL: http://app.hid.vm
+      SMTP_USER: 
+      SMTP_PASS:
+      APP_URL: https://app.hid.vm
```

Comment:
what is this for?

Reply:
Some emails sent out by the API have a link which needs to point to the app URL, which is configured here